### PR TITLE
Completed homework 2

### PR DESCRIPTION
**Pull request by m-tetiana**

### CODE REVIEW

File: js_basics/task5.1.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const radius = 10
+const area = Math.PI * radius  * radius
```
Comment (sttaran) [inline js_basics/task5.1.js:2]: use `Math.pow(radius, 2)` or `radius ** 2`. It will make your code more readable

File: js_basics/task5.3.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const height = 5
+const radius = 3
+const volume = Math.PI * radius * radius * height
```
Comment (sttaran) [inline js_basics/task5.3.js:3]: use `Math.pow(radius, 2)` or `radius ** 2`. It will make your code more readable

**Review by sttaran — CHANGES_REQUESTED** [review]

File: js_basics/task5.1.js (hunk 1 above)
Comment (m-tetiana) [reply js_basics/task5.1.js]: Fixed

File: js_basics/task5.3.js (hunk 2 above)
Comment (m-tetiana) [reply js_basics/task5.3.js]: Fixed

**Pull request merged**